src/main/scala/ch/openolitor/stammdaten/StammdatenCommandHandler.scala

```diff
@@ -549,13 +549,31 @@ trait StammdatenCommandHandler extends CommandHandler
     case UpdateEntityCommand(_, id: AboId, entity: AboVertriebsartModify) => idFactory => meta =>
       DB readOnly { implicit session =>
         //TODO: assemble text using gettext
-        val text = s"Vertriebsart angepasst. Abo Nr.: ${id.id}, Neu: ${entity.vertriebsartIdNeu}; Grund: ${entity.bemerkung}"
-        val pendenzEvent = addKundenPendenz(idFactory, meta, id, text)
-        Success(Seq(Some(EntityUpdateEvent(id, entity)), pendenzEvent).flatten)
+        val newLieferungen = stammdatenReadRepository.getLieferungen(entity.vertriebIdNeu)
+        stammdatenReadRepository.getById(depotlieferungAboMapping, id) match {
+          case Some(abo) =>
+            val text = s"Vertriebsart angepasst. Abo Nr.: ${id.id}, Neu: ${entity.vertriebsartIdNeu}; Grund: ${entity.bemerkung}"
+            var abscencesText = ""
+            var pendenzStatus: PendenzStatus = Erledigt
+            stammdatenReadRepository.getById(vertriebMapping, abo.vertriebId) match {
+              case Some(vertrieb) =>
+                stammdatenReadRepository.getLieferungen(vertrieb.id).filter(lOld => (lOld.datum isAfter DateTime.now) && (newLieferungen.count(l => l.datum == lOld.datum) == 0)) map { l =>
+                  if (stammdatenReadRepository.getAbwesenheit(abo.id, l.datum).length > 0) {
+                    abscencesText = s"; Bitte Absenzen prüfen!"
```

Comment:
Let's use `Abwesenheiten` instead of `Absenzen`. So it's the same everywhere.

Reply:
done